### PR TITLE
Fixed a Bug and added SQL Folder

### PR DESCRIPTION
**Pull request by RiyasAhamed21**

### CODE REVIEW

File: sql/MovieTicketApp.sql [hunk 1]
```diff
@@ -10,31 +10,54 @@ create table users(
...
-    status character varying(40) NOT NULL,
-    CONSTRAINT movies_pkey PRIMARY KEY (id)
-    screen character varying(40)  NOT NULL,
-    CONSTRAINT screen FOREIGN KEY (screen)
-        REFERENCES screens (name) MATCH SIMPLE
-    
-);
+create table screens(id serial primary key,
+					  name varchar(40) not null,
+					  status varchar(40) not null default 'INACTIVE',
```
Comment (csys-fresher-batch-2021) [inline sql/MovieTicketApp.sql:18]: Add check constraint for status column.

CHECK ( status in ('ACTIVE','INACTIVE') );

File: sql/MovieTicketApp.sql [hunk 2]
```diff
@@ -10,31 +10,54 @@ create table users(
...
+					   unique(name));
+					  
+insert into screens (name) 
+values('Screen 1'),
+('Screen 2'),
+('Screen 3'),
+('Screen 4');
 
 create table seat_types(
 	seat_type character varying(20) NOT NULL,
```
Comment (csys-fresher-batch-2021) [inline sql/MovieTicketApp.sql]: Unique constraint missing for seat type.

File: sql/MovieTicketApp.sql [hunk 3]
```diff
@@ -10,31 +10,54 @@ create table users(
...
+	id  serial NOT NULL ,
+	movie_name character varying(40) NOT NULL,
+    actor_name character varying(40) NOT NULL,
+    rating real NOT NULL,
+    status character varying(40) NOT NULL,
+    CONSTRAINT movies_pkey PRIMARY KEY (id),
+    screen character varying(40)  NOT NULL,
+    CONSTRAINT screen FOREIGN KEY (screen)
+        REFERENCES screens (name) MATCH SIMPLE
+
```
Comment (csys-fresher-batch-2021) [inline sql/MovieTicketApp.sql:66]: Add unique constraint for moviename - 
unique ( moviename, actor_name)

Comment (csys-fresher-batch-2021) [reply sql/MovieTicketApp.sql:66]: Add constraint for rating. 
CHECK ( rating >0 AND rating <=10 )

File: sql/MovieTicketApp.sql [hunk 4]
```diff
@@ -46,7 +69,7 @@ create table booking_details(
     status character varying(100) DEFAULT 'BOOKED',
```
Comment (csys-fresher-batch-2021) [inline sql/MovieTicketApp.sql]: Add check constraint for status

File: sql/MovieTicketApp.sql [hunk 5]
```diff
@@ -56,22 +79,6 @@ create table booking_details(
         REFERENCES show_times (show_time) 
 );
 
-create table admins(id serial primary key,
-				   user_name varchar(15) not null,
-				   password varchar(15) not null ,
-				   CONSTRAINT user_name UNIQUE (user_name),
-				   CONSTRAINT password UNIQUE(password));
```
Comment (csys-fresher-batch-2021) [inline sql/MovieTicketApp.sql:63]: remove unique constraint password.

**Review by ghost — CHANGES_REQUESTED** [review]
Added my code review comments.

**Review by ghost — APPROVED** [review]

**Pull request merged**